.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
     - TEST=python-sharded-and-javascript JS_SETUP=yes
   global:
     - REUSE_DB=true  # can't drop the citus DB since it's the postgres database
+    - JS_TEST_EXTENSIONS=icds_reports
```

Comment:
there's still an issue that if the build is being run from a fork PR then the ICDS code won't be available and this will fail. Not quite sure how to make it only run if the ICDS code is present